.github/mergeable.yml

```diff
@@ -1,30 +1,9 @@
 version: 2
 mergeable:
-  - when: pull_request.*, pull_request_review.*
-    name: 'PR title check'
-    validate:
-      - do: or
-        validate:
-          - do: title
-            must_include:
-              regex: '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\([a-z -]+\))?: [\w ]+$'
-              message: 'PR title needs to contain type'
-            must_exclude:
-              regex: "wip"
-          - do: title
-            must_include:
-              regex: 'bump [^ ]* from [^ ]* to [^ ]*'
-              message: 'PR title needs to comply with dependency bump (dependabot)'
-    pass:
-      - do: checks
-        status: 'success'
-        payload:
-          title: 'Mergeable Run have been Completed!'
-
   - when: pull_request.*, pull_request_review.*
     name: 'Approval check'
     validate:
       - do: approvals
         min:
           count: 1
-          message: 'Pull request must have at least 1 approvals'
+          message: 'Pull request must have at least 1 approval'
```

Comment:
I do not think we need this at all I think we can just delete the whole workflow since I have already set this rule in the actual repo settings. i.e. an admin needs to review and approve the PR before merge